lib/endpoints/class-wp-rest-comments-controller.php

```diff
@@ -318,6 +318,10 @@ public function create_item( $request ) {
 			return new WP_Error( 'rest_comment_exists', __( 'Cannot create existing comment.' ), array( 'status' => 400 ) );
 		}
 
+		if ( empty( $request['content'] ) || ! is_string( $request['content'] ) ) {
+			return new WP_Error( 'rest_comment_content_required', __( 'Missing comment content.' ), array( 'status' => 400 ) );
```

Comment:
Can we not just make this a `required` param in the schema here: https://github.com/WP-API/WP-API/blob/f73f3e8f7df3bebfc6e6c2476a4d6b2f1f2ec136/lib/endpoints/class-wp-rest-comments-controller.php#L854?